### ISSUE TITLE
Document reading timeseries from this repo

### ISSUE THREAD

**Issue by pieterprovoost** [iobis/edna-analysis-climate#1]
@silasprincipe Would you mind briefly documenting in the README how the data file in this repo can be read using R and/or Python, so people can easily get to the data during the workshop? Or just add a link to the data file from the README.

**Comment by silasprincipe** [comment]
Yes, I will make some updates to the code and after that I will document the data.
